scripts/build-alsa-tools.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2018 Intel Corporation. All rights reserved.
+
+# fail immediately on any errors
+set -e
+
+# Array of ALSA Git repository URLs.  Add or remove repositories as needed.
+declare -a REPOS=(
+    "https://github.com/alsa-project/alsa-lib.git"
+    "https://github.com/alsa-project/alsa-utils.git"
```

Comment:
Btw, the docker script for alsa-utils has built ALSA from thesofproject repos of alsa-lib and alsa-utils. I don't see an issue using upstream repos though, just as long we capture the commit-id to build in SOF side to ensure only versions we have validated are used.

Reply:
fixed